src/main/groovy/edu/oregonstate/mist/restpr/RESTPRConfiguration.groovy

```diff
@@ -0,0 +1,76 @@
+package edu.oregonstate.mist.restpr
+
+import io.dropwizard.Configuration
+import io.dropwizard.db.DataSourceFactory
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.google.common.collect.ImmutableMap
+import org.hibernate.validator.constraints.NotEmpty
+import javax.validation.Valid
+import javax.validation.constraints.NotNull
+import java.util.Collections
+import java.util.Map
+
+/**
+ * Created by georgecrary on 6/29/15.
+ */
+public class RESTPRConfiguration extends Configuration {
```

Comment:
i'm not sure it's necessary to have so much stuff in your configuration class.

in my sad api i was able to pare this down to 8 lines:
 https://github.com/osu-mist/sad-api/blob/d8add8c0c24fa0e08168ab9c8cb5d7370a31c4b9/src/main/groovy/edu/oregonstate/mist/sadapi/SadApplicationConfiguration.groovy